### DISCUSSION TITLE
Can i read specific b-values from my own multishell data?

### DISCUSSION THREAD

**Discussion by foxet** [dipy/dipy#2503] [3. Community Lounge]
Can I read specific range of b-values from my own multishell data, as I want to use diffusion data with b values <2000,

**Comment by skoudoro** [comment]
Hi @foxet,

To read your own bvals and bvecs:
```python
from dipy.io.gradients import read_bvals_bvecs

bvals, bvecs = read_bvals_bvecs('my_file.bvals', 'my_file.bvecs')
gtab = gradient_table(bvals, bvecs)
````

To select a range of bvals, look at the example in the following tutorial: https://dipy.org/documentation/1.4.1./examples_built/denoise_mppca/#example-denoise-mppca

Since this is not an issue, I move this thread in our discussion tab
